libs/blocks/card/card.js

```diff
@@ -32,7 +32,14 @@ const addWrapper = (el, section, cardType) => {
 
   if (prevGrid) return;
 
-  const upClass = getUpFromSectionMetadata(section);
+  let upClass = getUpFromSectionMetadata(section);
+  // Authored w/ a typed out number reference... 'two-up' vs. '2-up'
+  const list = ['two-up', 'three-up', 'four-up', 'five-up'];
+  const ixd = list.findIndex(i => i.includes(upClass));
```

Comment:
What does `ixd` stand for?

Reply:
I feel this should be `idx` short for index :)

Reply:
Oops. my bad on the typo. We'll clean that up in a future task. Thanks